### ISSUE TITLE
form fields missing a label on edit collection (sharing) page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6803]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the two` Select a role` boxes on the sharing tab of the `Edit Collection` page do not have a label and does not meet WCAG AA requirements

Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin, go to https://dev.nurax.samvera.org/dashboard/collections/bc386j44x/edit?locale=en, click the `Sharing` tab and run the Site Improve browser extension
![edit_collection_sharing_missing_label](https://github.com/samvera/hyrax/assets/32644254/8dbefb31-c400-4cd0-8bf9-73ceea7c2e7d)

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
